superset/assets/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -65,6 +81,10 @@ class ExploreResultsButton extends React.PureComponent {
   getQueryDuration() {
     return moment.duration(this.props.query.endDttm - this.props.query.startDttm).asSeconds();
   }
+  getInvalidColumns() {
+    const re = /^\w+$/;
```

Comment:
It might be better to use `/^[A-Za-z_]\w*/` here, to avoid `SELECT COUNT(1) AS 123`.

Reply:
Good call, added some tests for `'a', '1', '123'`.